### ISSUE TITLE
13B基座模型完全一样的参数下，training loss相比1代高很多

### ISSUE THREAD

**Issue by jeffchy** [baichuan-inc/Baichuan2#115]
hyper-params
global batch size 256
```
    --learning_rate 1.5e-4 \
    --weight_decay 0.01 \
    --adam_beta1 0.9 \
    --adam_beta2 0.95 \
    --max_grad_norm 1.0 
```
learning rate 在1e-4更小的时候也一样。
一代降低到1.9，二代停留在2.3。
请问是什么原因？
另外，我发现目前的zloss weight设置的是0，请问我继续pretrain或者sft的时候，需要打开zloss嘛？有什么建议？

**Comment by Lisennlp** [comment]
应该是baichuan2的词表变大了原因

**Comment by GuoxingY** [comment]
请问下训练的loss下降正常吗？我这边遇到训练loss不收敛的问题

**Comment by jeffchy** [comment]
发现加logits loss会对clm的trainingloss的下降有帮助

**Comment by GuoxingY** [comment]
> 发现加logits loss会对clm的trainingloss的下降有帮助

这个具体是指什么呀，能详细说说嘛

**Comment by jeffchy** [comment]
就是论文中的z loss
https://huggingface.co/baichuan-inc/Baichuan2-13B-Base/blob/main/configuration_baichuan.py#L27
可以去改这个进行设置。我这边任务贴近预训练，所以选用的是论文里面汇报的2e-4
